fe1-web/src/resources/strings.ts

```diff
@@ -291,7 +291,7 @@ namespace STRINGS {
     'The Roll Call is currently open and you as the organizer should start adding attendees by scanning their PoP tokens.';
   export const roll_call_open_attendee =
     'The Roll Call is currently open and you as an attendee should let the organizer scan your PoP token encoded in the QR Code below.';
-  export const roll_call_QRcode_text = 'To be scanned';
+  export const roll_call_qrcode_text = 'To be scanned to connect';
```

Comment:
I am not sure that this string is better than the precedent one, but I do not how to change it to be more explicit.

Reply:
I would try to make it either represent the intention of the attendee or the organizer. Since we have so far phrased most overlays from the perspective of the person scanning it, the intention of the organizer might be more appropriate so maybe something along the lines of "Scan to add attendee"?